AsyncDisplayKit/ASMapNode.h

```diff
@@ -0,0 +1,40 @@
+/* Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import <AsyncDisplayKit/AsyncDisplayKit.h>
+#import <MapKit/MapKit.h>
+@interface ASMapNode : ASControlNode
+- (instancetype)initWithCoordinate:(CLLocationCoordinate2D)coordinate NS_DESIGNATED_INITIALIZER;
+
+@property (nonatomic, readonly) ASImageNode *mapImage;
+@property (nonatomic, readonly) ASDisplayNode *liveMap;
+/**
+ Whether the map snapshot should turn into a MKMapView when tapped on. Defaults to YES.
+ */
+@property (nonatomic, assign) BOOL hasLiveMap;
```

Comment:
Can you make this default NO?  It's an important feature to have, but live maps are /very/ costly and it's also important to remove them once a scroll view starts moving again, so it should be an opt-in extra cost / functionality.

Any other name ideas?  Maybe becomesLiveMap, or even just showingLiveMap / similar.  Since it is an ASControlNode, it would be reasonable to expect developers to do addTarget:action ... and just enable the live map directly.  That's related to the need to be able to disable it too.